.golangci.yaml

```diff
@@ -1,24 +1,26 @@
+version: "2"
 run:
   build-tags:
     - requires_docker
-
 linters:
+  settings:
+    errcheck:
+      exclude-functions:
+        - (github.com/go-kit/log.Logger).Log
+  exclusions:
+    presets:
+      - comments
+      - common-false-positives
+      - legacy
+      - std-error-handling
+formatters:
   enable:
     - gci
     - gofmt
-
-linters-settings:
-  errcheck:
-    # Use exclude-functions instead of exclude, since the GitHub Action's caching is broken.
```

Comment:
Should we keep this comment?

Reply:
Sure. The migration command doesn't preserve comments and I wasn't sure if this was still relevant or not. If you're asking about it that's reason enough for me.